jodconverter-docker/build.gradle

```diff
@@ -36,6 +36,13 @@ dockerFile {
     }
 }
 
+docker {
+    registryCredentials {
+        username = System.getenv("DOCKER_USER")
```

Comment:
Can we make this optional, required only when publishing releases?
Right now this means you will always need to pass those environment variables when building the project

Reply:
@vierbergenlars I can build locally without those env var set ?

Reply:
Hmm, then the credentials are `null`, which is okay as long as there are no checks for that.